python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_base_group_chat.py

```diff
@@ -356,13 +363,22 @@ async def main() -> None:
         """
 
         # Create the first chat message if the task is a string or a chat message.
-        first_chat_message: ChatMessage | None = None
+        first_chat_message: ChatMessage | List[ChatMessage] | None = None
```

Comment:
Let's use `messages : List[ChatMessage] | None` here to simplify the code below.

Reply:
Thanks, simplified type hints to List[ChatMessage] or None